client/ayon_core/hosts/houdini/plugins/publish/collect_local_render_instances.py

```diff
@@ -0,0 +1,102 @@
+import os
+import pyblish.api
+
+
+class CollectLocalRenderInstances(pyblish.api.InstancePlugin):
+    """Collect instances for local render.
+
+    Agnostic Local Render Collector.
+    """
+
+    # this plugin runs after Collect Render Products
+    order = pyblish.api.CollectorOrder + 0.12
+    families = ["mantra_rop",
+                "karma_rop",
+                "redshift_rop",
+                "arnold_rop",
+                "vray_rop"]
+
+    hosts = ["houdini"]
+    label = "Collect local render instances"
+
+    def process(self, instance):
+
+        if instance.data["farm"]:
+            self.log.debug("Render on farm is enabled. "
+                           "Skipping local render collecting.")
+            return
+
+        # Create Instance for each AOV.
+        context = instance.context
+        expectedFiles = next(iter(instance.data["expectedFiles"]), {})
+
+        product_type = "render"  # is always render
+        product_group = "render{Task}{productName}".format(
```

Comment:
I guess we should use `get_product_name` for this and for product name of each aov.

Reply:
I've done it in this commit  https://github.com/ynput/ayon-core/pull/328/commits/90e2c1f1b5235e0d6fd408588a74f69f9cb7ac51 
let me know what do you think.